packages/fetch-and-compile/README.md

```diff
@@ -2,7 +2,11 @@
 
 This is used to obtain external verified sourced and compile them.
 
-### Usage
+Note: If you import this into your TS project, you may need to set `skipLibCheck` in your tsconfig due to an indirect dependency on @truffle/contract-schema.
```

Comment:
**Set** is ambiguous here, perhaps **enable**, or explicitly "set skipLibCheck to true"